src/Compilers/CSharp/Test/Emit/Emit/EmitMetadata.cs

```diff
@@ -2168,7 +2169,7 @@ private static void VerifyEmitWithNoResources(CSharpCompilation comp, Platform p
 
             using (var outputStream = new MemoryStream())
             {
-                var success = comp.WithOptions(options).Emit(outputStream).Success;
+                var success = comp.WithAssemblyName("EmitWithNoResourcesAllPlatforms_" + platform.ToString()).WithOptions(options).Emit(outputStream).Success;
```

Comment:
Changing that wasn't sufficient for VB tests, they kept failing randomly until I switched to CompileAndVerify helper.